packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -392,7 +392,7 @@ class HotRunner extends ResidentRunner {
     // We are now running from source.
     _runningFromSnapshot = false;
     final String launchPath = debuggingOptions.buildInfo.previewDart2
-        ? dillOutputPath ?? mainPath + '.dill'
+        ? mainPath + '.dill'
```

Comment:
Why did we remove support for `dillOutputPath` here?  This likely breaks dart-2 inside Google.

/cc @keertip

Reply:
Can you revert this change, the other changes can work w/o this change. Also, we need `dillOutputPath` for internal testing.

Reply:
See https://github.com/flutter/flutter/pull/15655#discussion_r176301237.

I believe I tested this to make sure latest version of Flutter works inside Google. Can you share logs that shed some more light why this is breaking?

Reply:
The latest version would work inside Google regardless because Google hasn't enabled Dart 2 yet (that's what @keertip has a pending CL to enable).  That said, you sound confident this was the right change.  You two should sync up tomorrow to make sure you're both on the same page 😄

Reply:
I tested it by having cl with @keertip's pending cl patched on top of flutter roll I did myself.

Reply:
Sorry for the spam then - nothing to see here...